### ISSUE TITLE
Incomplete jsrender.min.map file

### ISSUE THREAD

**Issue by yahyaerturan** [BorisMoore/jsrender#362]
I have an error in console about jsrender.min.map file:

```
DevTools failed to load SourceMap: Could not parse content for http://localhost/dist/js/jsrender.min.js.map: Unexpected token < in JSON at position 0
```

When I checked the file, it seems incomplete.

**Comment by BorisMoore** [comment]
Ah, thanks for calling this out. This is a repeat of a problem that occurred [once before (version v1.0.6)](https://github.com/BorisMoore/jsviews/issues/445#issuecomment-622160066) where the jsrender.min.map file is somehow getting truncated during the final build process. I'll probably need to release a new version (v1.0.9) to fix this on the CDNJS site.

But meantime you can get a valid jsrender.min.js.map file here:

https://www.jsviews.com/download/jsrender.min.js.map

(also linked from the https://www.jsviews.com/#download page).

Let me know if that version works correctly for you...

**Comment by BorisMoore** [comment]
This has been resolved in v1.0.9
